src/components/MapView/Analyser/index.tsx

```diff
@@ -360,6 +435,21 @@ function Analyser({ classes }: AnalyserProps) {
                 <Typography variant="body2">Run Analysis</Typography>
               </Button>
             )}
+            {analysisResult && (
+              <Button
+                className={classes.innerAnalysisButton}
+                onClick={shareAnalysis}
+                disabled={
+                  !!thresholdError || // if there is a threshold error
```

Comment:
Wasn't this copied from right above? We should put this large boolean expression in a constant